### PR TITLE
Update Create/Update methods to return the workflow

### PR DESCRIPTION
**Pull request by avivek**
Updated create and update org required workflow methods to return the required workflow object returned by the github api.
Merging this Fixes https://github.com/google/go-github/issues/2756

### CODE REVIEW

File: github/actions_required_workflows.go [hunk 1]
```diff
@@ -92,13 +92,18 @@ func (s *ActionsService) ListOrgRequiredWorkflows(ctx context.Context, org strin
...
-func (s *ActionsService) CreateRequiredWorkflow(ctx context.Context, org string, createRequiredWorkflowOptions *CreateUpdateRequiredWorkflowOptions) (*Response, error) {
+func (s *ActionsService) CreateRequiredWorkflow(ctx context.Context, org string, createRequiredWorkflowOptions *CreateUpdateRequiredWorkflowOptions) (*OrgRequiredWorkflow, *Response, error) {
 	url := fmt.Sprintf("orgs/%v/actions/required_workflows", org)
 	req, err := s.client.NewRequest("PUT", url, createRequiredWorkflowOptions)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
-	return s.client.Do(ctx, req, nil)
+	orgRequiredWorkflow := new(OrgRequiredWorkflow)
```
Comment (gmlewis) [inline github/actions_required_workflows.go:101]: Let's please add a blank line before this one.

File: github/actions_required_workflows.go [hunk 2]
```diff
@@ -92,13 +92,18 @@ func (s *ActionsService) ListOrgRequiredWorkflows(ctx context.Context, org strin
...
+	resp, err := s.client.Do(ctx, req, orgRequiredWorkflow)
+	if err != nil {
+		return nil, resp, err
+	}
+	return orgRequiredWorkflow, resp, nil
```
Comment (gmlewis) [inline github/actions_required_workflows.go:106]: Let's please add a blank line before this one.

File: github/actions_required_workflows.go [hunk 3]
```diff
@@ -124,13 +129,18 @@ func (s *ActionsService) GetRequiredWorkflowByID(ctx context.Context, owner stri
...
-func (s *ActionsService) UpdateRequiredWorkflow(ctx context.Context, org string, requiredWorkflowID int64, updateRequiredWorkflowOptions *CreateUpdateRequiredWorkflowOptions) (*Response, error) {
+func (s *ActionsService) UpdateRequiredWorkflow(ctx context.Context, org string, requiredWorkflowID int64, updateRequiredWorkflowOptions *CreateUpdateRequiredWorkflowOptions) (*OrgRequiredWorkflow, *Response, error) {
 	url := fmt.Sprintf("orgs/%v/actions/required_workflows/%v", org, requiredWorkflowID)
 	req, err := s.client.NewRequest("PATCH", url, updateRequiredWorkflowOptions)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
-	return s.client.Do(ctx, req, nil)
+	orgRequiredWorkflow := new(OrgRequiredWorkflow)
```
Comment (gmlewis) [inline github/actions_required_workflows.go:138]: Let's please add a blank line before this one.

File: github/actions_required_workflows.go [hunk 4]
```diff
@@ -124,13 +129,18 @@ func (s *ActionsService) GetRequiredWorkflowByID(ctx context.Context, owner stri
...
+	resp, err := s.client.Do(ctx, req, orgRequiredWorkflow)
+	if err != nil {
+		return nil, resp, err
+	}
+	return orgRequiredWorkflow, resp, nil
```
Comment (gmlewis) [inline github/actions_required_workflows.go:143]: Let's please add a blank line before this one.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @avivek !
Just a few minor tweaks, please.
Then we will be ready for a second LGTM+Approval from any other contributor to this repo before merging.

File: github/actions_required_workflows.go (hunk 1 above)
Comment (avivek) [reply github/actions_required_workflows.go:102]: Done.

File: github/actions_required_workflows.go (hunk 2 above)
Comment (avivek) [reply github/actions_required_workflows.go:108]: Done.

File: github/actions_required_workflows.go (hunk 3 above)
Comment (avivek) [reply github/actions_required_workflows.go:141]: Done

File: github/actions_required_workflows.go (hunk 4 above)
Comment (avivek) [reply github/actions_required_workflows.go:147]: Done.

**Comment by avivek** [comment]
Thanks for your quick review @gmlewis .
I have made the changes, please take a look now.

**Review by gmlewis — APPROVED** [review]
Thank you, @avivek !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**